types/amcharts/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for amCharts
```

Comment:
Do you mind checking what is the version number x.y this @types/amcharts supporting?

For example, if the @types/amcharts support 3.21.11, you can add like this
```
// Type definitions for amCharts 3.21
```

Reply:
Thanks, due to I am working with 3.21, I can only assume, it is working with 3.21.